[+] Add luxuanwen3 (#249)
one-among-us/data: people/luxuanwen3/page.en.md

---
name: Nini
info:
    alias: 鲵鲵,Nini
    location: Jiyuan, Henan Province, China
---

## Description

> **Nini** is a sweet transgender girl who has a keen interest in computers, experiences some anxiety, and is generally warm and friendly. She was taking medication and exogenous hormones on month-basis. Her avatar was created by the talented artist ディーバ/Deaver (@Deaver1229).

Reviewer: ```suggestion
**Nini** was a sweet trans girl with a keen interest in computers. She experienced some anxiety, and was generally warm and friendly.
She took medication and exogenous hormones on a monthly basis. Her avatar was created by the talented artist ディーバ/Deaver](https://x.com/Deaver1229).
```

> However, her parents had a lot of reservations about transgenderism, so she was closely monitored, and the only sister who can relate to her had difficulty communicating with her due to distance and academic commitments.

Reviewer: ```suggestion
However, her parents were resolutely opposed to her transgender identity and expression, so she was closely monitored.
The only sister who truly understood her found it difficult to communicate owing to distance and academic pressures.
```

Reviewer: 这里反对的更直接应该是跨性别认同与表达，在 One Among Us 的其他英文条目有过类似表述，可参考~

> Towards the end of July 2024, there was a extreme increase in the conflict between Nini and her parents. Her parents decided to stop her hormone replacement therapy and other medications, and they began to regard her as a rebel against orthodox. This was the first time she had considered taking her own life. After taking a significant amount of drugs and leaving her home, she was fortunate to be found by passersby after collapsing on the street. Subsequently, Nini was unable to access the drugs.

Reviewer: ```suggestion
Towards the end of July 2024, there was an extreme increase in conflict between Nini and her parents.
Her parents decided to stop her hormone replacement therapy and other medications, and they began to regard her as rebelling against orthodoxy.
This was the first time she had considered taking her own life.
After taking a significant amount of drugs and leaving her home, she was fortunate to be found by passersby after collapsing on the street.
Subsequently, Nini was unable to access the drugs.
```

> At 11:55 p.m. on August 4, 2024,Nini made the last decision. After posting her final tweet, she took an overdose of chemicals. Her close friends rushed around trying to bring her back. However,at 3 a.m. the next morning, Nini left us to pursue her own path and seek her own freedom.

Reviewer: ```suggestion
At 11:55 p.m. on August 4, 2024, Nini made her final decision.
After posting her last tweet, she took an overdose of chemicals.
Her close friends rushed to save her.
However, at 3 a.m. the next morning, Nini left us to pursue her own path and seek her freedom.
```

<PhotoScroll photos={[ '${path}/photos/i wish i could.png', ]} />

> Here I understand what is meant by glory:  
> the right to love without limits.  
> <p style="text-align: right;">― Albert Camus, Lyrical and Critical Essays</p>

> Nini is very warm to people, though this is not always evident in her outward demeanor. Rather, it is reflected in the pieces of her words that contain her equal and full love for everyone. She may call you "babe" and offer a hug. She will patiently explain what you don't know and try not to pass on her negativity to her friends.

Reviewer: ```suggestion
Nini was very warm to people, though this was not always evident in her outward demeanor.
Rather, it was reflected in the pieces of her words that conveyed her equal and full love for everyone.
She might call you "babe" and offer a hug.
She would patiently explain what you didn't know and try not to pass on her negativity to her friends.
```

> Hope you find your peace in silence.  
> Good night and wake up in a brand new world.  

> I (@Collapsar_ndn) haven't known Nini for very long, but I'm still affected by her positive energy, and I welcome all the other friends to contribute to the text and reminisce about our former friend,Nini.

Reviewer: ```suggestion
I (The author of this entry) hadn't known Nini for very long, but I was still deeply affected by her positive energy.
I invite all our friends to contribute to this entry and share memories of our dear friend, Nini.
```

> Thanks for your help to Nini. @awxs_2081573566 @Bolichara0 @N552AA

Reviewer: ```suggestion
Thanks for your help to Nini. @awxs_2081573566 @Bolichara0 @N552AA

Contributor for this entry: [Collapsar_ndn](https://twitter.com/Collapsar_ndn)
```
